CRM/Core/Invoke.php

```diff
@@ -150,8 +150,10 @@ public static function getItem($args) {
    * @throws \CRM_Core_Exception
    */
   public static function runItem($item) {
-    $ids = new CRM_Core_IDS();
-    $ids->check($item);
+    if (!defined('CIVICRM_IDS_DISABLE') || !CIVICRM_IDS_DISABLE) {
```

Comment:
without weighing in on the issue as a whole - this would be better

`CRM_Utils_Constant::value('CIVICRM_IDS_DISABLE', TRUE);`